documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -165,3 +165,42 @@ spec:
           key: cruise-control-log4j.properties
     # ...
 ----
+
+[[api-security-configuration]]
+[discrete]
+== API security
+
+The Cruise Control API is secured by default to protect the cluster against any potentially destructive Cruise Control operations.
+
+However, these security settings can be disabled.
+
+.An example of disabling Cruise Control API authorization.
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    config:
+      webserver.security.enable: false
+# ...
+----
+
+.An example of disabling Cruise Control API authentication.
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    config:
+       webserver.ssl.enable: false
+# ...
+----
```

Comment:
This is weird. Does Cruise Control really enable HTTP Basic authentication with `.ssl.enable` option? And Authorization with `security.enable`? If it is like than then I guess we have no choice, but this seems highly misleading.

Reply:
Sorry I mixed these up in the original description of the PR, it should be:
- webserver.security.enable  -> HTTP Basic authentication (which I am using for authorization)
- webserver.ssl.enable  ->  HTTPS authentication